Makefile

```diff
@@ -138,28 +148,53 @@ run: ${BUILD_DIR}/.runtime_deps_ready ${BUILD_DIR}/gecko/dist/b2g ${B2G_PROFILE_
 	startx ./b2g -no-remote -profile ${B2G_PROFILE_PATH} --screen ${RESOLUTION} -- /usr/bin/Xephyr \
         -title "RoninOS" -ac -br -noreset -screen ${RESOLUTION} \
                 -keybd ephyr,xkbmodel=evdev,xkblayout=${XKB_LAYOUT},xkbvariant=${XKB_VARIANT}
-	done
 	
 ${BUILD_DIR}/gaia/profile.tar.bz2: ${BUILD_DIR}/gaia/profile
 	cd ${BUILD_DIR} && \
 	tar --directory gaia/profile -cjf  ${BUILD_DIR}/gaia/profile.tar.bz2 `ls gaia/profile`
 		
+
+clean-package:
+	if [ -d ${BUILD_DIR}/package ]; then rm -rf ${BUILD_DIR}/package; fi
+
 ###### Build deb package
 # We need gaia profile, gecko build and a few scripts
-package: build ${BUILD_DIR}/gaia/profile.tar.bz2
-	if [ ! -d ${BUILD_DIR}/package/opt/b2g ]; then mkdir -p ${BUILD_DIR}/package/opt/b2g; fi
-	if [ ! -d ${BUILD_DIR}/package/DEBIAN ]; then mkdir -p ${BUILD_DIR}/package/DEBIAN; fi
-	if [ ! -d ${BUILD_DIR}/package/usr/share/xsessions ]; then mkdir -p ${BUILD_DIR}/package/usr/share/xsessions; fi
-
-	cp ${BUILD_DIR}/gaia/profile.tar.bz2 ${BUILD_DIR}/package/opt/b2g/ 
-	tar --directory ${BUILD_DIR}/package/opt/b2g -xjf ${BUILD_DIR}/gecko/dist/b2g-${GECKO_VERSION}.0a1.en-US.linux-x86_64.tar.bz2
-	cp ${mkfile_dir}/launch.sh ${BUILD_DIR}/package/opt/b2g/
-	cp ${mkfile_dir}/session.sh ${BUILD_DIR}/package/opt/b2g/
-	cp ${mkfile_dir}/b2g.desktop ${BUILD_DIR}/package/usr/share/xsessions/
-	sed 's/DATETIME/${DATETIME}/' ${mkfile_dir}/control > ${BUILD_DIR}/package/DEBIAN/control
-	cd ${BUILD_DIR} && \
-	fakeroot dpkg-checkbuilddeps package/DEBIAN/control && \
-	fakeroot dpkg-deb -b package b2g_${GECKO_VERSION}.0a1-${DATETIME}_amd64.deb
+package: clean-package ${BUILD_DIR}/gaia/profile.tar.bz2 ${BUILD_DIR}/gecko/dist/b2g
+	if [ ! -d ${PACKAGE_DIR} ]; then mkdir -p ${PACKAGE_DIR}; fi
+	if [ ! -d ${PACKAGE_DIR}/wetty ]; then mkdir -p ${PACKAGE_DIR}/wetty; fi
+	
+	# Prepare package dir
+	tar --directory ${PACKAGE_DIR} -xjf ${BUILD_DIR}/gecko/dist/b2g-${GECKO_VERSION}.0a1.en-US.linux-x86_64.tar.bz2
+	cp ${BUILD_DIR}/gaia/profile.tar.bz2 ${PACKAGE_DIR} 
+	cp ${mkfile_dir}/launch.sh ${PACKAGE_DIR}
+	cp ${mkfile_dir}/session.sh ${PACKAGE_DIR}
+	cp ${mkfile_dir}/ronin.desktop ${PACKAGE_DIR}
+
+	cd ${WETTY_SOURCE_DIR} && cp -r app.js package.json public ${PACKAGE_DIR}/wetty
+
+	# Generate ${PACKAGE_DIR}/debian folder
+	cd ${PACKAGE_DIR} && dh_make -n -s -y -e tech@phoxygen.com
```

Comment:
I always had the feeling `dh_make` should be run once, then that the debian folders should be committed and used by other dh-script, WDYT?